CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 2.45.0-rc.0 / 2023-06-05
+
+This release is a LTS (Long-Term Support) release of Prometheus and will
+receive security, documentation and bugfix patches for at least 6 months.
+Please read more about our LTS release cycle at
+<https://prometheus.io/docs/introduction/release-cycle/>.
+
```

Comment:
Did you manage to go through commits to find interesting changes? It's tedious and prone to errors, but no other way ): 

I think we missed https://github.com/prometheus/prometheus/commit/a16b876a05faef651f6fd4f3dd71a32a996ba367 for sure (just remember it merged recently)